### DISCUSSION TITLE
sdlf-codebuild

### DISCUSSION THREAD

**Discussion by ghost** [aws-solutions-library-samples/data-lakes-on-aws#60] [General]
When I start the self-foundations for the 2nd time and do the git push it shows me a failed, what could be failing? how can I solve that? as this prevents me from troubleshooting the sdlf-team creation for the sdlf-team-stagea / sdlf-team-repositories

**Comment by kukushking** [comment]
Hi @wrojasprojc could you show the error message you are receiving, please?
